### DISCUSSION TITLE
Thread Support for Sonoff Zigbee dongles

### DISCUSSION THREAD

**Discussion by alexruffell** [Koenkk/Z-Stack-firmware#431] [General]
I've been following the development of all functionality related to Matter in Home Assistant. I have 2 Sonoff Zigbee dongles (P version with TI chip) and was wondering when there would be a firmware than enables the Thread protocol so I could use one of the dongles for Zigbee and one for Thread (unless there is strong evidence that running both on one radio is better).

**Comment by Pete1979** [comment]
Also looking for an answer to this. I would like to run both Thread and Zigbee on the same stick tho, since i only have one CC2652R (zzh!) stick :)

**Comment by Flavien** [comment]
The answer is never as this is out of scope of this project. @Koenkk had mentioned on another thread that this firmware is meant to be Zigbee only. But feel free to fork the project and start a new one with Thread support.

**Comment by Hedda** [comment]
>  Sonoff Zigbee dongles (P version with TI chip) and was wondering when there would be a firmware than enables the Thread protocol

As mentioned, question abut Thread is out-of-scope and off-topic here, however to point you in the right direct suggest you check out this discussion which includes some information about OpenThread Border Router firmware for TI with Spinel  + HDLC UART serial interface:   

* https://github.com/home-assistant/addons-development/issues/38

Anyway, read there you need an OTBR (OpenThread Border Router) RCP firmware build for CC2652P with Spinel  + HDLC + UART enabled for Texas Instruments, but do not ask more about OpenThread RCP firmware here, instead suggest that you open a new discussion about that elsewhere, maybe in the Home Assistant community forums -> https://community.home-assistant.io/ (or maybe in https://github.com/home-assistant/addons if you plan on using their [Thread integration](https://www.home-assistant.io/integrations/thread/) in combination with their [OpenThread Border Router Add-on](https://github.com/home-assistant/addons/tree/master/openthread_border_router) in Home Assistant).

They will also hold a live stream tomorrow where you can ask -> https://www.home-assistant.io/blog/2024/01/05/matter-livestream/

You can post questions in the comments sections -> https://community.home-assistant.io/t/the-state-of-matter-livestream/667009

Many questions and answers already posted previously too -> https://community.home-assistant.io/t/nabu-casa-at-the-matter-member-meeting/650164/

> I could use one of the dongles for Zigbee and one for Thread (unless there is strong evidence that running both on one radio is better).

FYI, the CC2652/CC1352 series do not support Thread and Zigbee concurrent multi-PAN so there is no point in asking about that regardless.

PS: Again, note that has nothing to do with Z-Stack, Zigbee2MQTT/Z2M, or zigbee-herdsman as this and those projects are all Zigbee only.
